docs/console-virtual-terminal-sequences.md

```diff
@@ -107,6 +107,25 @@ The following commands control the visibility of the cursor and its blinking sta
 > [!TIP]
 > The enable sequences end in a lowercase H character (`h`) and the disable sequences end in a lowercase L character (`l`).
 
+## <span id="Cursor_Shape"></span><span id="cursor_shape"></span><span id="CURSOR_SHAPE"></span>Cursor Shape
+
+
+The following commands control and allow for customization of the cursor shape.
+
+
+| Sequence | Code | Description | Behavior |
+|---------------|---------|------------------------------|---------------------------|
+| ESC \[ 0 SP q | DECSCUSR | User Shape | Default cursor shape configured by the user |
```

Comment:
Elsewhere on this page, the Description column shows how the control sequence is defined by the industry, and the Behavior column shows how Windows Terminal implements it. Did DEC define DECSCUSR with parameter 0 as activating the cursor shape preferred by the user, or is that specific to Windows Terminal?

Page 5-127 of <http://boundlessterminals.com/manuals/VT520-VT525ProgrammerInformation.pdf> lists "0, 1 or none" as selecting the "Blink Block (Default)" cursor style. That suggests to me that the number 1 means this style specifically, and 0 or none means the default style, which in those DEC terminals is hardcoded rather than a user preference. So the pull request looks OK in that respect.

Does Windows Terminal support DECSCUSR without a parameter? Perhaps that too should be documented here, something like the paragraph in another section of this page: <https://github.com/MicrosoftDocs/Console-Docs/blob/e1d1841c175e4f5b3c2f469d8515a975c83a28d9/docs/console-virtual-terminal-sequences.md?plain=1#L171>